### DISCUSSION TITLE
Add GitHub Actions and Bots to Keploy Repositories

### DISCUSSION THREAD

**Discussion by Ayush7614** [keploy/keploy#86] [Ideas]
We need to have project-specific bots later (Especially for Python, and JS-based projects) to track code quality, builds and all. Please have a look and let me know what you think. You can also add more bot integrations that we can have right here.

This issue would be used to track the integration of GitHub Bots across Circuitverse repositories.

1 . Prosebot [Integration](https://github.com/apps/prosebot)
*  Checks for grammatical errors in the Docs. It is specially for docs
*  Analyzes spelling, prose and inclusive verbiage in Markdown and Text Files.
*  Provides an easy way to analyze documentation changes and ensure a consistency throughout

2 . Stale Bot  [Integration](https://github.com/apps/stale)
* Closes Issues and Pull Requests if not attended to in a specified period of time
* Maintains the overall sanctity of the issues and Pull Requests tab.
*  Keeps a check on spam/unnecessary Issues and PRs.

3 . Greet Bot [Integration](https://github.com/actions/first-interaction)
* Greet First Time Contributors

4 . Request - Info [Integration](https://github.com/apps/request-info)
* Request for the information if the issues/PR Content body is empty
*  Help us in Spam / invalid issues and PRs

We can also add many bots i have more specific bots and i can also add in the future

**Comment by Ayush7614** [comment]
![image](https://user-images.githubusercontent.com/67006255/159469474-a3e1bdbd-4f46-4314-b80e-534b84ad8648.png)
Specially for Automation we use this GHA

**Comment by Ayush7614** [comment]
cc @nehagup

**Comment by slayerjain** [comment]
Thanks for posting this @Ayush7614 - this sounds awesome. It would definitely make sense to add have this for all the repos and not just the python and js sdk repos.
I also think we should be a bit careful with the #4 (Request - Info) integration because many fields could be optional and could also unnecessarily spam users/contributors.
